source/annex/macro.rst

```diff
@@ -234,25 +72,23 @@ are adjusted and in the second run the AEEI coefficients are adjusted. This cali
 :math:`{GROW\_corr}_{y, r}` and the AEEI coefficients :math:`{aeei}_{r, y, s}` all stay below :math:`10^{-5}`.
 
 Iterating between MESSAGE and MACRO
-----
+-----------------------------------
 
 Exchanged parameters
-~~~~
+~~~~~~~~~~~~~~~~~~~~
 MESSAGE and MACRO exchange demand levels of the six commercial servcie demand categories represented in MESSAGE, their corresponding prices as well as total energy system costs including
 trade effects of energy commodities and carbon permits (if any explicit mititgation effort sharing regime is implemented).
 
 Convergence criterion
```

Comment:
Section on convergence is not specific to global model (part of GAMS code) and therefore should be moved to MACRO part of MESSAGEix documentation.

Reply:
Perhaps the whole section here could be transferred to the MACRO docs?